src/components/views/auth/AuthBody.js

```diff
@@ -17,10 +17,26 @@ limitations under the License.
 'use strict';
 
 import React from 'react';
+import PropTypes from 'prop-types';
+
```

Comment:
ooi, why the blank line?

Reply:
hmm yeah, part of me thought they should be separate but probably only because the others have capitals - probably unnecessary.